app/components/works/work_type_modal_component.html.erb

```diff
@@ -14,39 +14,45 @@
       </div>
 
       <div class="modal-body">
-        <form method="get" action="/collections/1/work/new" class="mb-4" data-work-type-target="form">
+        <form method="<%= form_method %>" action="/collections/1/work/new" class="mb-4" data-work-type-modal-target="form" data-work-type-target="form" data-work-type-edit-target="form" <% unless turbo? %> data-turbo=false<% end %>>
```

Comment:
The work type form had to become way more complicated to handle this use case (accounting for using PUT and working around turbo).